### PR TITLE
https://redmine.legimi.com/issues/21633

### PR DESCRIPTION
**Pull request by blazej-kosmowski**

### CODE REVIEW

File: ContextMenu.Droid/ContextMenuScrollViewRenderer.cs [hunk 1]
```diff
@@ -148,7 +148,7 @@ private async void OnScrollToRequestedNew(object sender, ScrollToRequestedEventA
 				await Task.Delay(TimeSpan.FromMilliseconds(1));
 				cycle++;
 
-				if (_isDisposed)
+				if (CanHandle() == false)
```
Comment (rhaly) [inline ContextMenu.Droid/ContextMenuScrollViewRenderer.cs:149]: I would check this also before entering while loop

File: ContextMenu.Droid/ContextMenuViewRenderer.cs [hunk 2]
```diff
@@ -149,6 +149,9 @@ private async void OnScrollToRequestedNew(object sender, ScrollToRequestedEventA
 				await Task.Delay(TimeSpan.FromMilliseconds(1));
 				cycle++;
 
+				if (CanHandle() == false)
```
Comment (rhaly) [inline ContextMenu.Droid/ContextMenuViewRenderer.cs:150]: I would check this also before entering while loop

**Review by rhaly — CHANGES_REQUESTED** [review]

File: ContextMenu.Droid/ContextMenuScrollViewRenderer.cs (hunk 1 above)
Comment (blazej-kosmowski) [reply ContextMenu.Droid/ContextMenuScrollViewRenderer.cs:149]: Done

File: ContextMenu.Droid/ContextMenuViewRenderer.cs (hunk 2 above)
Comment (blazej-kosmowski) [reply ContextMenu.Droid/ContextMenuViewRenderer.cs:150]: Done

**Review by rhaly — APPROVED** [review]

**Pull request closed without merging**